lib/fittings/swagger_render.js

```diff
@@ -6,6 +6,7 @@ var injector = require('../../index.js').injector;
 var _ = injector.get('_');
 
 module.exports = function create(fittingDef) {
+    injector = require('../../index.js').injector;
```

Comment:
The difference is in workflows we are using simpleWrapper to mock an injectable and what's happening is the swagger_render takes this injectable and exports the function with it.  The Profiles test then continues to use this "stale" injectable.
By adding in this line we are essentially refreshing the injectable to make sure if there were any changes the exported function takes them into account.

@zyoung51 can correct me if my understanding is wrong or if he wants to add more :)